src/static_data/get_static_data.go

```diff
@@ -0,0 +1,217 @@
+	filePath := filepath.Join(folderDir, filename)
+	file, err := os.Create(filePath)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+
+	// Copy the file from the response body to the local file
+	_, err = io.Copy(file, resp.Body)
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("Downloaded: %s\n", filePath)
+	return nil
+}
+
+func main() {
+	folders := []Folder{}
+
+	// Create folder instances
+	bgptest1 := Folder{
+		FolderName: "bgptest1",
+		Urls: []string{
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211204.1845.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211204.1900.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211204.1915.bz2",
+		},
+	}
+	folders = append(folders, bgptest1)
+
+	bgptest2 := Folder{
+		FolderName: "bgptest2",
+		Urls: []string{
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211204.1545.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211204.1600.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211204.1615.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211204.1630.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211204.1645.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211204.1700.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211204.1715.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211204.1730.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211204.1745.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211204.1800.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211204.1815.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211204.1830.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211204.1845.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211204.1900.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211204.1915.bz2",
+		},
+	}
+	folders = append(folders, bgptest2)
+
+	bgptest3 := Folder{
+		FolderName: "bgptest3",
+		Urls: []string{
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211209.1530.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211209.1545.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211209.1600.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211209.1615.bz2",
+		},
+	}
+	folders = append(folders, bgptest3)
+
+	bgptest4 := Folder{
+		FolderName: "bgptest4",
+		Urls: []string{
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211220.1615.bz2",
```

Comment:
If you want to -- just a suggestion. Each of the URL's follows a similar patten (the only thing that is different is the time stamp) you could get creative and make this a variable and append the timestamp that you are watching to shrink this down. Def not a concern but would be a cleaner design.

Reply:
this is also an easy pull request for the future!